hawc/apps/lit/templates/lit/overview.html

```diff
@@ -130,7 +130,13 @@ <h2 class="d-inline-block">Literature Review</h2>
     </div>
     <div class="col-md-4">
       <div class="lit-review-box box-shadow" style="width: 100%;">
-        <p class="h4 m-0">Tagtree</p>
+        <p class="h4 m-0">Tagtree
```

Comment:
@casey1173 what do you think of using bootstrap badges? 
![image](https://user-images.githubusercontent.com/999952/221927399-c76509ec-d96e-404b-b30e-e14b04dabdb5.png)

Reply:
Love it!!

Reply:
Those look great